docs/api/api_metrics.rst

```diff
@@ -0,0 +1,32 @@
+Metrics of HOI
+--------------
+
+:py:mod:`hoi.metrics`:
+
+.. currentmodule:: hoi.metrics
+
+.. .. automodule:: hoi.metrics
+..    :no-members:
+..    :no-inherited-members:
+
+.. toctree::
+..    :maxdepth: 2
+..    :caption: Contents:
+
+..    OinfoZeroLag <generated/metricsOfHoi/hoi.metrics.oinfo_zerolag>
```

Comment:
Why do you need those lines? `OinfoZeroLag <generated/metricsOfHoi/hoi.metrics.oinfo_zerolag>`